src/App/Fossa/Config/Common.hs

```diff
@@ -171,40 +179,64 @@ metadataOpts =
     <*> parsePolicyOptions
     <*> many (strOption (applyFossaStyle <> long "project-label" <> stringToHelpDoc "Assign up to 5 labels to the project"))
     <*> optional releaseGroupMetadataOpts
+
+deprecateConfigurableProjectMetadata :: Has Diagnostics sig m => ProjectMetadata -> m ProjectMetadata
+deprecateConfigurableProjectMetadata ProjectMetadata{..} = do
+  when
+    ( isJust projectTitle
+        || isJust projectUrl
+        || isJust projectJiraKey
+        || isJust projectLink
+        || isJust projectPolicy
+        || isJust projectTeam
+        || not (Prelude.null projectLabel)
+    )
+    $ warn deprecationMessage
+
+  pure $ ProjectMetadata Nothing Nothing Nothing Nothing projectTeam Nothing [] projectReleaseGroup
   where
-    policy :: Parser Policy
-    policy = PolicyName <$> (strOption (applyFossaStyle <> long "policy" <> helpDoc policyHelp))
-
-    policyId :: Parser Policy
-    policyId =
-      PolicyId
-        <$> ( option
-                (readMWithError "failed to parse --policy-id, expecting int")
-                (applyFossaStyle <> long "policy-id" <> helpDoc policyIdHelp)
-            )
-
-    parsePolicyOptions :: Parser (Maybe Policy)
-    parsePolicyOptions = optional (policy <|> policyId) -- For Parsers '<|>' tries every alternative and fails if they all succeed.
-    titleHelp :: Maybe (Doc AnsiStyle)
-    titleHelp =
-      Just . formatDoc $
+    deprecationMessage :: Text
+    deprecationMessage =
+      renderIt $
         vsep
-          [ "The title of the FOSSA project"
-          , boldItalicized "Default: " <> "The project name"
+          [ annotate (color Red) "Project options (--title, --project-url, --jira-project-key, --link, --policy, and --project-label) have been deprecated for this command."
+          , "Refer to `fossa project` subcommands to interact with FOSSA projects."
```

Comment:
Yup they're deprecated, we want users to use `project edit` to modify these project configurations.

Reply:
Can `fossa project edit` also create new projects? 

There is an org-wide setting [to require all projects to have a policy](https://github.com/fossas/FOSSA/pull/10171). If there weren't a way from the CLI to both create a project and set the policy in one step, I think this means a customer couldn't make a new project. Or would have to use Quick Import then have to switch to the CLI? I'm not totally sure exactly how it would work. 

In the future, do you also see deprecating these directives for `fossa analyze` in `.fossa.yml`?